foundation-extensions/profile/profile-api-references.md

```diff
@@ -83,6 +83,15 @@ public static func updateUserAttributes(attributeDict: [String: Any])
 
 You want to update `username, usertype` of a user obtained in the log in page :
 
+**Swift**
```

Comment:
I see a few styling issues in this file, here are a few pointers to bring this in line to the other extensions:
- the API name should be used as header outside of the platform tab, following by the generic description, then platform tabs with Syntax and Examples each.
- the API names should be ordered alphabetically.
- For Syntax and Examples, we use bold \*\*String\*\* notation, so let's remove any unnecessary headers, e.g. `#### **Syntax**` should be `**Syntax**`
- For the programming language we usually use h3, e.g. `### Swift`
- Let's use Objective-C for all occurrences